app/styles/globals.css

```diff
@@ -75,3 +75,8 @@ html {
 main {
   height: 100%;
 }
+
+
+.myBtn:hover .nextIcon{
+  fill: black !important;
+}
```

Comment:
Please avoid adding styles to the globals.css, instead use the respective component css file